### ISSUE TITLE
Support Exif 3.0

### ISSUE THREAD

**Issue by aentwist** [darktable-org/darktable#14809]
**Is your feature request related to a problem? Please describe.**

I think Exif 3.0 came out recently. See the [press release](https://www.iptc.org/news/exif-3-0-released-featuring-utf-8-support/).

Not sure what work if any is required to support it. Figured it can be tracked using this issue.

**Comment by aentwist** [comment]
I'd consider this extremely low priority at the moment.

**Comment by kmilos** [comment]
Depends on Exiv2 supporting it: https://github.com/Exiv2/exiv2/issues/2659

**Comment by kmilos** [comment]
From dt's POV it would be interesting to support some of the new DAM tags, but not clear to me ATM how we can tell if we're "developing" vs "editing"...
